lib/l10n_utils/__init__.py

```diff
@@ -127,6 +130,9 @@ def render(request, template, context=None, ftl_files=None, activation_files=Non
     name_prefix = request.path_info.split("/", 2)[1]
```

Comment:
Reading the `render()` method in full might be easier than just looking at the changed lines here